### ISSUE TITLE
Suggestion for "Improvised Explosive" - ANFO (Ammonium Nitrate Fuel Oil)

### ISSUE THREAD

**Issue by Frostben** [yogstation13/yogstation#1870]
A while back I saw an issue where it was discussed that the current IED is a piece of shit, shittier than its improvised nature would suggest, and there was some discussion of increasing its power or having an alternative IED. 

My suggestion is to include the off the shelf and insanely easy to produce blasting agent ANFO which has quite the history as a go to terrorist weapon. As the title explains ANFO is the mixture of Ammonium Nitrate (fertilizer) and Fuel Oil, fuel oil being broad in the sense that diesel, gasoline, or welding fuel could be used as an equally efficient substitute. From some research I've learned that the ratio for ANFO is around 4% fuel oil and around 96% ammonium nitrate give or take.

The explosion itself can be quite powerful especially in large quantities. An example of this would be the terrorist attack in Oklahoma 1995 that destroyed an entire federal building killing hundreds of people. This was achieved with 1-3 tons of ANFO in a van in the parking garage. Even 50lbs of ANFO can be quite destructive https://www.youtube.com/watch?v=UWscaO49MCk
An example of a 1 tons worth of ANFO - https://www.youtube.com/watch?v=saU3ftuG7CQ

Now obviously this would be ludicrous to allow people to make that amount in game even 50lbs worth of ANFO would wreck a huge part of the station. So why not downsize it to maybe 5lbs of anfo or less in a storage container ie backpack, satchel, briefcase, or even a duffelbag? ANFO also cannot be simply lit on fire to detonate it, it requires a blasting charge (in this case maybe the current IED strapped to the container?) in order to detonate it. Tell me what you think! Is this a fucking stupid idea that will never make it into code even though the ingredients are easily available on station? Would the amount of blasting agent have to be reduced to a single pound for it to even be thought about?

TL;DR: A bag with a small Ammonium Nitrate Fuel Oil mixture with a normal welder fuel IED attached to it for an improved and deadlier version of our current IED. Crafting Recipe - Bag/Satchel/Briefcase, 50u of Welder Fuel, 50u of Ammonium Nitrate, and an Improvised IED. Would be 5x stronger than the current IED (give or take this is just to give an idea of the explosive potential).

**Comment by X-TheDark** [comment]
While the idea itself is pretty great, including "good" explosives that can easily be made by anyone would be a bad idea, as far as gameplay goes (obviously, discussion is welcome).

There is a good reason bombs (TTVs) have so many rules attached to them:
1. Science only;
2. Non-antag station bombing is bannable;
3. Even antag station bombing is heavily discouraged;

There are also balance concerns, since bombs do have a good stun on them and in general are quite strong, as far as damage goes (IEDs notwithstanding). In addition, it always kind of sucks to be stunned/killed by a bomb that's outside your viewrange (In this case, about 2 screens away, since medium blast is really powerful).

**Comment by Frostben** [comment]
Yeah that's quite true. Eventually an antag could just break into Toxins and make a TTV himself if he has the know-how. Maybe I should switch my brainpower to something else besides explosives for now.

I'll leave this issue open for another day or so in case some other folks want to talk about it and give their input.

**Comment by AdamElTablawy** [comment]
Making IED's vary in blast power depending on what reagent they use is something I can do, but remember that an IED will never be better than a welderbomb or slightly higher in strength.

**Comment by Frostben** [comment]
@AdamElTablawy That sounds like an even better idea then this whole issue.
